tests/compliance/date/test_date_compliance.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for extension interface compliance, inherited from pandas.
+
+See:
+https://github.com/pandas-dev/pandas/blob/main/pandas/tests/extension/decimal/test_decimal.py
+and
+https://github.com/pandas-dev/pandas/blob/main/pandas/tests/extension/test_period.py
+"""
+
+import datetime
+
+from pandas.tests.extension import base
+
+import db_dtypes
+
+
+class TestDtype(base.BaseDtypeTests):
+    pass
+
+
+class TestInterface(base.BaseInterfaceTests):
+    pass
+
+
+class TestConstructors(base.BaseConstructorsTests):
+    pass
+
+
+class TestReshaping(base.BaseReshapingTests):
+    pass
+
+
+class TestGetitem(base.BaseGetitemTests):
+    def test_take_na_value_other_date(self):
```

Comment:
It would be helpful to have an additional comment in the test to clarify the expected behaviour

Reply:
👍  Would have been nice if pandas had a bit more of an explanatory comment, but I'll try and deduce why this test was added in https://github.com/pandas-dev/pandas/pull/20814. Perhaps it's not even necessary in general?

Reply:
Removed, as most other extension dtype tests don't have it.